plone/app/z3cform/templates/contentbrowser_display.pt

```diff
@@ -0,0 +1,46 @@
+<span class="contentbrowser-widget ${python:normalizeString(view.field.__class__.__name__)}-field"
+      id="form-widgets-${python:view.__name__}"
+      tal:define="
+        plone_view nocall:context/@@plone;
+        normalizeString python:plone_view.normalizeString;
+        items python: view.items();
+      "
+      tal:condition="items"
+      i18n:domain="plone"
+>
+  <div>
+    <ul>
```

Comment:
We might want to use a `pat-sortable` class here?
Note: `pat-sortable` from Patternslib would need to make use of `MutationObserver` which it doesn't, currently.